cmake/iotjs.cmake

```diff
@@ -497,6 +497,9 @@ set(IOTJS_PUBLIC_HEADERS
 
 # Configure the libiotjs
 set(TARGET_LIB_IOTJS libiotjs)
+# Configure the libiotjs.a
+set(TARGET_STATIC_IOTJS libiotjs)
```

Comment:
I don't tnink it's necessary to add another alias.

Reply:
it's not anymore